docs/manual-installation.md

```diff
@@ -27,24 +53,51 @@ Open vSwitch >= 2.8.0 userspace daemon `ovs-vswitchd` and `ovsdb-server` should
 and `ClusterRoleBinding` sections in the [Deployment yaml](/build/yamls/antrea.yml) to configure
 Kubernetes RBAC to do it.
 
-2. Create the kubeconfig file that contains the tokens or certificates of ServiceAccount or user created in the above
-step. See [Configure Access to Multiple Clusters](
+2. Create the kubeconfig file for K8s APIServer access that contains the server endpoint and the token or certificate
+of ServiceAccount or user created in the above step. See [Configure Access to Multiple Clusters](
 https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) for more information.
 
-3. Create the antrea-agent config file, see [Configuration](configuration.md) for details.
+3. Create the kubeconfig file for `antrea-controller` APIServer access that contains the server endpoint and the token
+of ServiceAccount created in the above step.
+```
+TOKEN=$(kubectl get secrets -n kube-system -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='antrea-agent')].data.token}"|base64 --decode)
+
+cat >antrea.conf <<EOF
+apiVersion: v1
+kind: Config
+preferences: {}
+current-context: antrea-agent@kubernetes
+
+clusters:
+- cluster:
+    server: https://<ANTREA_CONTROLLER_ENDPOINT>
+    insecure-skip-tls-verify: true
+  name: kubernetes
+
+users:
+- name: antrea-agent
+  user:
+    token: $TOKEN
+
+contexts:
+- context:
+    cluster: kubernetes
+    user: antrea-agent
+  name: antrea-agent@kubernetes
+EOF
+```
+
+4. Create the antrea-agent config file, see [Configuration](configuration.md) for details.
 ```
 cat >antrea-agent.conf <<EOF
 clientConnection:
   kubeconfig: <PATH_TO_KUBE_CONF>
+antreaClientConnection:
+  kubeconfig: <PATH_TO_ANTREA_KUBE_CONF>
```

Comment:
maybe make it clear that `PATH_TO_ANTREA_KUBE_CONF` is the file from the previous step (3)

Reply:
Done